src/components/probe/prober/http/index.ts

```diff
@@ -57,11 +56,6 @@ export class HTTPProber extends BaseProber {
       responses.push(
         // eslint-disable-next-line no-await-in-loop
         await httpRequest({
-          isVerbose: getContext().flags.verbose,
```

Comment:
is this intentional? because it is added in the previous PR https://github.com/hyperjumptech/monika/pull/1271/files

<img width="1287" alt="Screenshot 2024-04-16 at 17 23 33" src="https://github.com/hyperjumptech/monika/assets/5974862/6d750e6a-556b-4bb8-9a32-2f1317c2cdf8">

Reply:
Yes. Context is a global variable, no need to pass it through a parameter, you can access it directly.

Reply:
maybe @syamsudotdev need to see this too to confirm

Reply:
> Yes. Context is a global variable, no need to pass it through a parameter, you can access it directly.
I second this for fewer function params since this PR still tests the functions correctly.